java/arcs/core/host/HandleConnectionSpec.kt

```diff
@@ -0,0 +1,10 @@
+package arcs.core.host
+
+/**
+ * Represents a use of a [Handle] by a [Particle].
+ */
+data class HandleConnectionSpec(
+    val connectionName: String,
+    val usedHandle: HandleSpec,
```

Comment:
imo the `using` and `used` prefixes in the names are more confusing than helpful.

Reply:
Suggestion? Since the spec is bidirectional, from/to don't make much sense. I guess I could just use 'handleSpec' and 'particleSpec' and leave it at that.